yarn-project/prover-node/src/bond/config.ts

```diff
@@ -0,0 +1,25 @@
+import { type ConfigMappingsType, bigintConfigHelper, getConfigFromMappings } from '@aztec/foundation/config';
+
+export type ProverBondManagerConfig = {
+  proverMinimumStakeAmount: bigint;
+  proverTargetStakeAmount?: bigint;
```

Comment:
Bit late to be nitty picking, but "StakeAmount" implies funds are subject to slashing and from what i can tell those are just "deposits" in the escrow contract. i.e. they're only at stake once a prover's quote has been accepted.

Reply:
Good point. WDYT about `proverMinimumEscrowAmount`?

Reply:
descriptive, ship it!